### PR TITLE
doc/radosgw: edit sections

### PR DESCRIPTION
**Pull request by zdover23**
Edit the following sections in doc/radosgw/admin.rst:

* Remove a User
* Remove a Subuser

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

File: doc/radosgw/admin.rst [hunk 1]
```diff
@@ -197,41 +197,44 @@ in a command of the following form:
...
 
-When you remove a user, the user and subuser are removed from the system.
-However, you may remove just the subuser if you wish. To remove a user (and
-subuser), specify ``user rm`` and the user ID. ::
+The act of removing a user removes both the user and any subuser associated
+with the user from the system. Remember that it is possible to remove a subuser
+without removing the user with which it is associated.   
 
-	radosgw-admin user rm --uid=johndoe
+To remove a user and any subuser associated with it, use the ``user rm``
```
Comment (anthonyeleven) [inline doc/radosgw/admin.rst:204]: plural, as in plural zed alpha

File: doc/radosgw/admin.rst [hunk 2]
```diff
@@ -197,41 +197,44 @@ in a command of the following form:
...
+  with the UID.
 
 
 Remove a Subuser
 ----------------
 
-When you remove a sub user, you are removing access to the Swift interface. 
-The user will remain in the system. To remove the subuser, specify 
-``subuser rm`` and the subuser ID. ::
+Removing a subuser removes access to the Swift interface. The user associated
```
Comment (anthonyeleven) [inline doc/radosgw/admin.rst:224]: ... or S3

**Review by anthonyeleven — APPROVED** [review]
See comments

File: doc/radosgw/admin.rst (hunk 1 above)
Comment (zdover23) [reply doc/radosgw/admin.rst:204]: Accepted.

File: doc/radosgw/admin.rst [hunk 3]
```diff
@@ -197,41 +197,44 @@ in a command of the following form:
 Remove a User
 -------------
 
-When you remove a user, the user and subuser are removed from the system.
-However, you may remove just the subuser if you wish. To remove a user (and
-subuser), specify ``user rm`` and the user ID. ::
+The act of removing a user removes both the user and any subuser associated
```
[the remark this answers is not in the document]
Comment (zdover23) [reply doc/radosgw/admin.rst:200]: Accepted.

File: doc/radosgw/admin.rst (hunk 2 above)
Comment (zdover23) [reply doc/radosgw/admin.rst]: Accepted.

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/55027 - Reef backport
https://github.com/ceph/ceph/pull/55028 - Quincy backport

**Pull request merged**